src/zoid/buttons/util.js

```diff
@@ -50,8 +82,8 @@ export function createVenmoExperiment() : Experiment | void {
     const isEligibleForVenmo = fundingEligibility && fundingEligibility[FUNDING.VENMO] && fundingEligibility[FUNDING.VENMO].eligible;
 
     // exclude buyers who are not eligible
-    // exclude integrations using enable-funding=venmo
-    if (!isEligibleForVenmo || isEnableFundingVenmo) {
+    // exclude non-desktop integrations using enable-funding=venmo
+    if (!isEligibleForVenmo || (isEnableFundingVenmo && isDevice())) {
```

Comment:
What does `enable-funding=venmo` mean for Venmo Desktop? It appears that these integrations using would `enable-funding=venmo` would only get venmo desktop if they are bucketed into the experiment.

Reply:
Yep that is correct. Even though Venmo Desktop & Venmo App-Switch share literally everything (funding source, payment flow, query params, etc.) we're treating them as two separate things in regards to the roll out. 

My assumption is that there are merchants/implementations that use `enable-funding=venmo` to surface the Venmo App-Switch button/flow on mobile, but  we don't want to make Venmo Desktop available to initially. So until we change the experiment value to something other than 0, the only way Venmo Desktop will be available is by manually adding a page to experiment  via `window.localStorage.setItem('enable_venmo_desktop', true);`. 

Ideally, I think how this ultimately will/should work is that if a merchant or implementation wants to show venmo as funding source (either through natural eligibility or explicitly setting `enable-funding=venmo`), we do and then make a determination on whether to do initialize the QRCode or App-Switch flow based on their device.